.github/mergify.yml

```diff
@@ -12,9 +12,41 @@ queue_rules:
 pull_request_rules:
   - name: automerge to the base branch with label automerge and branch protection passing
     conditions:
-      - "#approved-reviews-by>=1"
+      # Pull requests targeting `main` branch that have `A:automerge` label
       - base=main
       - label=A:automerge
+      # with 2 reviews and 0 changes requested left
+      - "#approved-reviews-by>=2"
```

Comment:
Can we keep this at 1? Tbh this is mostly useful for simple PRs that don't require 2 approvals

Reply:
sure